py/torch_tensorrt/__init__.py

```diff
@@ -91,7 +91,7 @@ def _find_lib(name, paths):
 from torch_tensorrt import logging
 from torch_tensorrt._Input import Input
 from torch_tensorrt._Device import Device
-from torch_tensorrt._TRTModuleNext import TRTModuleNext
+from torch_tensorrt.dynamo import TorchTensorRTModule
```

Comment:
This gives users the option to _either_ do `from torch_tensorrt.dynamo import TorchTensorRTModule` or `from torch_tensorrt import TorchTensorRTModule`. We can remove the latter by removing this line.

Reply:
Causes a circular import issue - removing this line in favor of `from torch_tensorrt.dynamo import TorchTensorRTModule`